lib/osf-components/addon/components/osf-layout/registries-side-nav/label/styles.scss

```diff
@@ -0,0 +1,15 @@
+.LabelWrapper {
+    justify-content: space-between;
+    display: inline-flex;
+    width: 100%;
+    padding-left: 30px;
+    height: 100%;
+}
+
+.Label {
+    font-weight: 700;
```

Comment:
Is it possible to combine `.Label` and `.Count` under a single class so we don't repeat the same style? (or will it work to have `font-weight` under `.LabelWrapper` instead?)